certif/app/serializers/application.js

```diff
@@ -1 +1,5 @@
-export { default } from '@ember-data/serializer/json-api';
+import JSONAPISerializer from '@ember-data/serializer/json-api';
+
+export default class ApplicationSerializer extends JSONAPISerializer {
+  extractErrors = 1;
```

Comment:
```suggestion
  // This bypasses extractErrors emberData behavior which relies on the property being a function.
  // <Behaviour will be entirely shifted when moving to v5 anyways>
  extractErrors = 1;
```

Reply:
j'ai mis `<Behaviour will be entirely shifted when moving to v5 anyways>` comme placeholder. Il faut trouver la formulation qui a le plus de sens pour nous.